generators/boilerplate/index.js

```diff
@@ -6,6 +6,14 @@ module.exports = generators.Base.extend({
   constructor: function () {
     generators.Base.apply(this, arguments);
 
+    // destination path
+    this.option('path', {
```

Comment:
I was thinking about using `destinationPath` the name as the method but it could quickly become confusing. Agree for `projectRoot`.